Source/Core/Core/PowerPC/CachedInterpreter/CachedInterpreter.cpp

```diff
@@ -274,13 +291,19 @@ void CachedInterpreter::Jit(u32 address)
       if (idle_loop)
         m_code.emplace_back(CheckIdle, js.blockStart);
       if (endblock)
+      {
         m_code.emplace_back(EndBlock, js.downcountAmount);
+        m_code.emplace_back(UpdateNumLoadStoreInstructions, js.numLoadStoreInst);
```

Comment:
Are any games known to use these counters? If not, two calls at the end of every block is steep price to pay. Have you considered replicating the MCR check that is in the JIT? It's definitely a hack, but it seems justifiable if the feature is only exercised by test code.

Reply:
No games use them as far as I know. (We should've gotten reports about things breaking on ARM by now otherwise.)

I was a bit unsure if I should add that hack here or not when I was working on the code. But I'd like to have input from one more person before switching it over.

Reply:
Have you measured the performance impact?

Reply:
I tried it now, and it actually seems to be running one FPS faster with this change than without. I'm going to assume it's just a random variation. (The scene I used was the file select screen in Anubis II, which runs at about 33 FPS for me. The scene has 3D graphics but is otherwise relatively light.)

Reply:
IMO it's not worth porting the speed hack to the interpreter, then.

Reply:
Actually, never mind about no games using them. Harry Potter and the Prisoner of Azkaban uses them according to cdace9d, and that game has indeed been known to be broken in JitArm64.